### PR TITLE
Update Disnake to support Forum Channels

### PR DESCRIPTION
**Pull request by SimplyMika**
Also fixes missing column in database insert. Not sure why this wasn't caught earlier.

Updates Disnake to 2.6.0 (latest version as of this PR).

### CODE REVIEW

File: cogs/mod.py [hunk 1]
```diff
@@ -186,7 +186,7 @@ def when(self, record):
 # ripped from RoboDanny
 class BannedMember(commands.Converter):
 	async def convert(self, ctx, argument):
-		ban_list = await ctx.guild.bans()
+		ban_list = await ctx.guild.bans(limit = None).flatten()
```
Comment (onerandomusername) [inline cogs/mod.py]: futureproof:
```suggestion
		ban_list = list(await ctx.guild.bans(limit=None))
```

File: cogs/mod.py [hunk 2]
```diff
@@ -363,16 +363,16 @@ async def tempban(self, ctx, member: Union[disnake.Member, BannedMember], amount
...
+						'VALUES ($1, $2, $3, $4, $5, $6, $7, $8, $9)',
+						ctx.guild.id, member.id, ctx.author.id, 'BAN', now, duration, reason, self._craft_user_data(member), False
 					)
 				except UniqueViolationError:
 					# this *should* never happen but I'd rather leave it in
 					raise commands.CommandError('Member is already tempbanned.')
 
 				try:
-					await ctx.guild.ban(member, delete_message_days=0, reason=reason)
+					await ctx.guild.ban(member, clean_history_duration=0, reason=reason)
```
Comment (onerandomusername) [inline cogs/mod.py:375]: should be fixed as of 2.6.1 but this is a good idea nonetheless

File: requirements.txt [hunk 3]
```diff
@@ -1,4 +1,4 @@
-disnake==2.3.0
+disnake==2.6.0
```
Comment (onerandomusername) [inline requirements.txt]: something should probably be added to the contributing guide about requiring message content intent within the configuration now

**Review by onerandomusername — COMMENTED** [review]
untested, code review only

File: requirements.txt (hunk 3 above)
Comment (onerandomusername) [inline requirements.txt]:
```suggestion
disnake==2.6.1
```

**Comment by Run1e** [comment]
Thanks for the PR, I will try it and merge it when I get on my workstation again.

The reason for the missing database column insert is that there's a mismatch between my manual changes to the live database and migrate.sql. The `completed` field is supposed to have a default value of `FALSE` but I just forgot to add that to migrate.sql.

The changes done to the sql query can be undone and migrate.sql fixed instead.

**Comment by onerandomusername** [comment]
just fyi, disnake 2.7.0 will be released tomorrow, along with 2.6.2

**Pull request merged**